wallet/src/plan/planner.rs

```diff
@@ -68,14 +69,11 @@ impl<R: RngCore + CryptoRng> Planner<R> {
 
     /// Add a fee to the transaction plan.
     ///
-    /// Calling this function more than once will add to the fee, not replace it.
+    /// This function should be called once.
```

Comment:
We can't add to the existing fee, since there is a single fee field per transaction

(I could have changed this to allow addition when the fee asset IDs match, but that seems unnecessary and this is simpler)

Reply:
Yeah, I don't think there's a good use case for incrementally adding fees.